### ISSUE TITLE
Error under error icon is not announced by screen reader

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#183]
![image](https://user-images.githubusercontent.com/2320302/119193291-47c49d00-ba36-11eb-8f0b-7227cf2b7dde.png)

**Comment by msftedad** [comment]
Hi team, Any update on this issue?
